primitives/float/get_hardfloat.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
Can we add a documentation section under the standard library that tells people that they need to run this command to get the floating-point libraries?

Also, this will be tricky to make work with the Compiler release (`cargo install calyx`) which essentially bundle all the Verilog files and save them on the user's computer. It might make sense to expose a subcommand to the `calyx` binary to download these kinds of dependencies. Can you open an issue about this part (and add the documentation to this PR)?